x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -802,6 +810,110 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     return this.esClient.search<SafeEndpointEvent>(query);
   }
 
+  public async fetchValueListMetaData(interval: number) {
+    if (this.esClient === undefined || this.esClient === null) {
+      throw Error('elasticsearch client is unavailable: cannot retrieve diagnostic alerts');
+    }
+
+    const listQuery: SearchRequest = {
+      expand_wildcards: ['open' as const, 'hidden' as const],
+      index: '.lists-*',
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      body: {
+        aggs: {
+          total_value_list_count: {
+            cardinality: {
+              field: 'name',
+            },
+          },
+          type_breakdown: {
+            terms: {
+              field: 'type',
+            },
+          },
+        },
+      },
+    };
+    const itemQuery: SearchRequest = {
+      expand_wildcards: ['open' as const, 'hidden' as const],
+      index: '.items-*',
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      body: {
+        aggs: {
+          value_list_item_count: {
+            terms: {
```

Comment:
We only get the top 10 largest lists with this agg. Can we retrieve more without overloading the telemetry request when we send the data? 

@peluja1012 @jethr0null what might be a good maximum number of value lists to retrieve telemetry on?

Reply:
Also, IP range value lists support values in 2 distinct notations: CIDR (`127.0.0.1/16`) and dash (`127.0.0.1-127.0.0.5`). In the internal doc structure they get converted to `"ip_range": "127.0.0.1/16"` and
```
"ip_range": {
  "gte": "127.0.0.1",
  "lte": "127.0.0.5"
}
```
respectively. Items within a list do not all have to use the same notation. If we can get stats about how many items in each IP range value list use each notation, that would be fantastic as well, though I don't know if there's a way to do it aside from actually retrieving all the items and processing them in memory which probably isn't worth it IMO. If we can't do it, that's fine, but it's worth checking on.

Reply:
>@peluja1012 @jethr0null what might be a good maximum number of value lists to retrieve telemetry on?

I'm not sure how many lists users have in their environments on average, but we could maybe start with 20 and update in future releases based on this information.